src/containers/CallIn/CallIn.test.js

```diff
@@ -1,33 +0,0 @@
-import React from "react";
-import { MemoryRouter } from "react-router-dom";
-import { createMemoryHistory } from "history";
-import { render, fireEvent, waitFor } from "@testing-library/react";
-import { logCall as logCallAmplitude } from "../../util/amplitude";
-import "@testing-library/jest-dom/extend-expect";
-
-import { CallIn } from "./CallIn";
-
-jest.mock("../../util/axios-api");
-jest.mock("../../util/amplitude");
-
-describe("CallIn", () => {
-  test("logCall() invoked when I Called is clicked", async () => {
```

Comment:
how about updating the test to confirm that the redirect is made if `didCall` is true?